stages/eib_image

```diff
@@ -420,8 +420,43 @@ EOF
         umount "${DEPLOY}"/{boot,dev,img,proc,sys}
         rm -f "${DEPLOY}"/img
 
-        # Standard endless system - grub, shim, all that kind of thing
-        cp -r "${DEPLOY}"/usr/lib/efi_binaries/EFI "${ESP}"
+        # Install EFI binaries.
+        #
+        # In EOS6, the the binaries in `/usr/lib/efi_binaries` got there from
+        # `/boot/efi` in the initial Debian compose via eos-ostree-builder:
+        # <https://github.com/endlessm/eos-ostree-builder/blob/master/stages/eob_ostree#L76>
+        #
+        # The eos-build-meta GRUB elements install the GRUB x86_64-efi binary
+        # to same path ready to go in the ESP.
+        #
+        # For EOS7, we copy the Shim binary signed with Microsoft's certificate
+        # from Git. It's not gonna change in the short term as we would need
+        # to request a new signature from Microsoft.
+        #
+        # I also copied the mokmanager binary signed with Endless' certificate
+        # from an EOS6 system. This could instead be built in eos-build-meta
+        # and signed via the Endless secure-boot signing API.
```

Comment:
This comment seems more appropriate within the commit message adding shim to eos-build-meta.

Overall do we need to include any of these changes in the image builder here? Can we just have the ostree build set up /usr/lib/efi_binaries/EFI in the currently expected way and have the existing image builder code run?

Reply:
Simplified the changes in 38c0c375df6550c5d7f95bd95ca272d91b2061da